src/components/routes/RouteFromPath.js

```diff
@@ -1,7 +1,10 @@
 import { Route } from 'react-router-dom';
 import { node } from 'prop-types';
 
-const RouteFromPath = ({ component, ...route }) => <Route {...route}>{component}</Route>;
+
+const RouteFromPath = ({ component, ...route }) => (
+  <Route {...route}> {component} </Route>
+);
```

Comment:
```suggestion
const RouteFromPath = ({ component, ...route }) => <Route {...route}>{component}</Route>;
```

To keep this file unchanged